src/js/BookReader.js

```diff
@@ -2681,6 +2681,16 @@ BookReader.prototype.updateFromParams = function(params) {
     }
   }
 
+
+  // process /search
+  // @deprecated for urlMode 'history'
+  // Continues to work for urlMode 'hash'
+  if (this.enableSearch && 'undefined' != typeof(params.search)) {
+    if (this.searchTerm != params.search) {
```

Comment:
SUGGESTION: Prefer `!==`?

Reply:
Yeah, I resurrected this code from before my refactor and didn't verify that it wasn't type coercing. I'm guessing not, since this should always be treated as a string, even if it is only comprised of numbers.